go-update/src/main/java/org/reactome/release/goupdate/GoTermsUpdater.java

```diff
@@ -307,6 +328,21 @@ else if (goTermsFromFile.get(goID).containsKey(GoUpdateConstants.IS_OBSOLETE) &&
 		return mainOutput;
 	}
 
+	protected static Map<GKSchemaAttribute, Integer> getReferrerCounts(GKInstance inst) throws Exception
+	{
+		Map<GKSchemaAttribute, Integer> referrersCount = new HashMap<GKSchemaAttribute, Integer>();
+		for (GKSchemaAttribute attrib : (Collection<GKSchemaAttribute>)inst.getSchemClass().getReferers())
```

Comment:
Same thing I mentioned up above -- does inst.getSchemClass().getReferers() return GKSchemaAttributes?

Reply:
Nevermind... I can see from the type that it does